.swiftlint.yml

```diff
@@ -1,43 +0,0 @@
-swiftlint_version: 0.25.0
```

Comment:
Keep swiftlint?

Reply:
I'll do after I finish the implementation